### ISSUE TITLE
Linkedin r_fullprofile scope login failed 

### ISSUE THREAD

**Issue by chatzich** [singingwolfboy/flask-dance#333]
Hello,
I try to create a login process with Linkedin OAuth service.
I have followed the instructions and retrieve the proper data of r_liteprofile
`linkedin_bp = make_linkedin_blueprint(scope=["r_liteprofile"])`
the user data provided by `r_liteprofile` is not enough for my purpose and 
I need the `r_fullprofile` scope data but unfortunately when I changed the scope to
`linkedin_bp = make_linkedin_blueprint(scope=["r_fullprofile"])`

![Screenshot at 2020-03-12 00-13-02](https://user-images.githubusercontent.com/189659/102134907-d3227c80-3e5f-11eb-81dc-ed81514cd1d1.png)

**Comment by singingwolfboy** [comment]
That looks expected to me. You can't just suddenly change your scope and expect everything to work without asking users to re-authorize. They need to grant permissions for the updated scopes.

**Comment by chatzich** [comment]
> That looks expected to me. You can't just suddenly change your scope and expect everything to work without asking users to re-authorize. They need to grant permissions for the updated scopes.

But how? I thought that there was an app permissions problem

**Comment by singingwolfboy** [comment]
@chatzich I think maybe we're misunderstanding each other. What are the exact steps to reproduce this problem?

**Comment by chatzich** [comment]
> @chatzich I think maybe we're misunderstanding each other. What are the exact steps to reproduce this problem?

Yes I think that my application does not have the r_fullprofile scope permission that's why I am getting the error if the app had the permission the linked api service should ask the user if he agrees to provide the new permissions
